more/free-programming-cheatsheets.md

```diff
@@ -116,6 +116,8 @@
 ### Data Science
 
 * [Cheatsheets for Data Scientists](https://www.datacamp.com/community/data-science-cheatsheets) - Datacamp (PDF)
+* [Data Science Cheat Sheets](https://medium.com/analytics-vidhya/data-science-cheat-sheets-109ddcb1aca8) - Surya Gutta (HTML, PDF)
+* [Data Science Cheatsheets](https://goodjob0823.github.io/Data-Science-Cheat-Sheets) - goodjob0823 (HTML, PDF)
```

Comment:
this is a list of links to cheat sheets. Our list is not a list of lists. romove this.